apps/common/models/i-casework-getcase.js

```diff
@@ -29,9 +30,11 @@ module.exports = class DocumentModel extends Model {
   }
 
   fetch() {
-    const options = this.requestConfig({});
-    options.qs = this.prepare();
-    options.method = 'GET';
-    return this.request(options);
+    return this.prepare().then(qs => {
+      const options = this.requestConfig({});
+      options.qs = qs;
+      options.method = 'GET';
+      return this.request(options);
```

Comment:
because this.request is returned and is an async func already we don't need to adjust anything handling this fetch function as it was treated as async to begin with